rotten_tomatoes/correlation_analysis.py

```diff
@@ -0,0 +1,26 @@
+import seaborn as sns
+import pandas as pd
+import numpy as np
+
+class CorrelationAnalysis:
+    
+    def __init__(self, d):
```

Comment:
You might want to add some types here. Something like 
`def __init__(self, d: pd.DataFrame) -> None:`

And everything is going to need docstrings.

Reply:
Updated to include type and docstring